src/test/resources/arquillian-tests/build.gradle

```diff
@@ -83,9 +83,7 @@ dependencies {
     
     libertyRuntime group: runtimeGroup, name: kernelArtifactId, version: runtimeVersion
 
-    if(JavaVersion.current().isJava11()) {
-        implementation "javax.annotation:javax.annotation-api:1.3.2"
-    }
+    implementation "javax.annotation:javax.annotation-api:1.3.2"
```

Comment:
Does this not negatively impact the runs with Java 8?

Reply:
No. Java 8 has consistently passed with no problems.